dev/Telemetry/RuntimeProfiler.cpp

```diff
@@ -73,6 +73,73 @@ namespace RuntimeProfiler {
             }
         }
         
+        //  This function returns the start of the string presentation of
+        //  given number, it assumes the buffer passed has a sufficiently
+        //  large buffer to accomodate the number.
+        static PWSTR ConstructNumber(_In_ UINT uNumber, _Out_ PWSTR pszEnd)
```

Comment:
This code base uses modern C++. Please use std::wstring or std::wstring_view as appropriate for PWSTR. This also makes it unnecessary to use SAL annotations.

Reply:
I also think this is equivalent to a StringCchPrintfW or wcstol function. Please use standard ones instead of writing them by hand.